toxav/ring_buffer.h

```diff
@@ -0,0 +1,18 @@
+#ifndef RING_BUFFER_H
```

Comment:
@GrayHatter It's literally a name of the data type. I don't see a point in renaming it for this special case.